lib/sles4sap.pm

```diff
@@ -166,7 +166,8 @@ sub prepare_profile {
 
     if ($has_saptune) {
         assert_script_run "saptune daemon start";
-        assert_script_run "saptune solution verify $profile";
+        script_run "saptune solution verify $profile";
+        record_info("saptune solution verify $profile", "Check the status! assert_script_run has been removed to workaround saptune_V2 issues...");
```

Comment:
WDYT about getting the return code from the `script_run` and only calling `record_info` when the return code is true with a message indicating that `saptune solution verify` returned warnings or errors?

Reply:
Yes, that possible, and maybe more a `record_soft_failure` in that case? Could be easier to see in the whole `record_info`'s :)

Reply:
I do like the `record_soft_failure` in this case, as the `saptune verify` command is basically failing, this is a better way not to ignore the error.